### PR TITLE
[intro.refs] Add normative references to LIA-1 and IEEE 754.

### LINKED ISSUE #237: What is LIA-1?
Footnotes 210-212, 214-217, 219-223, in 18.3.2.4 [numeric.limits.members], refers to "LIA-1". This name is not mentioned anywhere else in the entire standard. Maybe the reference in footnote 205, which mentions "ISO/IEC 10967-1 Language independent arithmetic - Part 1", should say something like "hereafter referred to as LIA-1"?

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #237.

### CODE REVIEW

**Comment by zygoloid** [comment]
Adding normative references here is clearly correct, but also a normative change, so I'd like explicit CWG approval for this.

**Comment by jensmaurer** [comment]
I've uploaded the updated [intro.refs] page to the CWG wiki for Kona:
http://wiki.edg.com/pub/Wg21kona2017/CoreWorkingGroup/lia-ieee.pdf

**Comment by jensmaurer** [comment]
CWG suggested to add the year in p5 introducing the LIA-1 abbreviation and otherwise approved the change.

**Comment by jensmaurer** [comment]
Rebased and implemented CWG guidance.

File: source/support.tex [hunk 1]
```diff
@@ -730,9 +730,9 @@
 \begin{itemdescr}
 \pnum
 Measure of the maximum rounding error.\footnote{Rounding error is described in
-ISO/IEC 10967-1 Language independent arithmetic - Part 1
-Section 5.2.8 and
-Annex A Rationale Section A.5.2.8 - Rounding constants.}
+LIA-1
+Section 5.2.4 and
+Annex C Rationale Section C.5.2.4 --- Rounding constants.}
```
Comment (zygoloid) [inline source/support.tex]: The name of this section is "Rounding and rounding constants", not merely "Rounding constants".

Comment (jensmaurer) [inline source/support.tex]: Fixed.

**Pull request merged**